README.md

```diff
@@ -9,6 +9,7 @@ configuration options for production-grade deployments.
 * 5.6-onbuild (uses apache)
 * 7.2-onbuild (uses apache)
 * 7.3-onbuild (uses apache)
+* 7.3-cli
```

Comment:
the `-onbuild` postfix should be added, as there are "ONBUILD" directives in the docker file, indicating this is a specific type of image. Much like "abstract class"

Reply:
added